src/app/inventory/reducer.ts

```diff
@@ -54,10 +54,10 @@ export interface InventoryState {
   readonly readOnly: boolean;
 
   /**
-   * a JSON-encoded API profile response. if this is present,
+   * A JSON-encoded API profile response. if this is present,
    * we use it instead of talking to the Bungie API
```

Comment:
```suggestion
   * A JSON-encoded API profile response. If this is present,
   * we use it instead of talking to the Bungie API.
```

Reply:
This is also no longer JSON-encoded, but a proper JS object.